torch/csrc/jit/mobile/register_mobile_ops.cpp

```diff
@@ -315,4 +326,17 @@ static auto registry0 = torch::RegisterOperators().op(
   torch::RegisterOperators::options().catchAllKernel(
   []() {
   })
+).op(
+  "_aten::append.Tensor(Tensor self) -> void",
```

Comment:
Yep, schema in this file may be wrong. @smessmer Just curious: if we are using names only, would the schema be necessary in mobile? It looks like schema is required for boxed registration, but it's not checked in the registration itself. Could you confirm if a correct schema is required for registrations in this file. If it is, how to know what's the correct schema?

Reply:
if you register multiple ops with the same operator name + overload name but mismatching schemas, the operator library will throw an error. This didn't happen here because `_aten::append.Tensor` is allowed to mismatch with `aten::append.Tensor`, they're different ops.